src/platform/CMakeLists.txt

```diff
@@ -27,16 +27,11 @@ else()
     set(SOURCES ${SOURCES} inline.c platform_posix.c storage_posix.c cgroup.c datapath_unix.c)
     if(CX_PLATFORM STREQUAL "linux" AND NOT CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
         set(SOURCES ${SOURCES} datapath_linux.c datapath_epoll.c)
-        if ((${UBUNTU_VERSION} STREQUAL "20.04") OR
-            (${CMAKE_TARGET_ARCHITECTURE} STREQUAL "arm64") OR
-            (${CMAKE_TARGET_ARCHITECTURE} STREQUAL "arm") OR
-            (${CMAKE_SYSTEM_PROCESSOR} STREQUAL "arm") OR
-            (ANDROID) OR
-            (QUIC_CARGO_BUILD))
-            set(SOURCES ${SOURCES} datapath_xplat.c datapath_raw_dummy.c)
-        else()
+        if ("${UBUNTU_VERSION}" STREQUAL "22.04")
```

Comment:
I turned the condition around so that XDP will light up only on Ubuntu 22.04.
But I think it would be better to check for XDP support in a distribution agnostic way.
cc @wfurt

Reply:
Could you add below? (not sure right syntax). only allowing Ubuntu22.04 and x64 (amd64) for now.
`( ${CMAKE_TARGET_ARCHITECTURE} STREQUAL "x64") OR ${CMAKE_SYSTEM_PROCESSOR} STREQUAL "x64") ) AND !(ANDROID) AND !(QUIC_CARGO_BUILD)`

Reply:
I generally feel putting version numbers here is bad idea. Versions do change and how about Debian and other distributions? 

My suggestion would be to parametrize the build ... something like `--enable-xdp` and put the logic into pipeline part e.g. it would work in CI but the make logic would not have all the glory details. 

And it would be much more friendly to any other distribution IMHO.

Reply:
I like @wfurt idea, @ami-GS are you ok with me adding a new argument to the build?

Reply:
I'm fine with that model.